app/views/evidence_hub/index.html.erb

```diff
@@ -44,13 +44,7 @@
                 <%= render 'shared/evaluation_types', type: "#{document.evidence_type}" %>
               </div>
               <div class="search-results__data-type">
-                <ul class="evidence-types">
-                  <% document.data_type_keys.each do |data_type| %>
-                    <li class="evidence-types__item data-types__<%= "#{data_type.downcase}" %>">
-                      <%= render 'shared/evidence_types', icon: document.data_type_icon(data_type), text: data_type %>
-                    </li>
-                  <% end %>
-                </ul>
+                <%= render 'shared/data_types', presenter: document %>
```

Comment:
Actually I approved but taking a second look. The shared directory is already huge:

```
⤷  tree app/views/shared/
app/views/shared/
├── _country_list.html.erb
├── _evaluation_types.html.erb
├── _evidence_hub_hero.html.erb
├── _evidence_types.html.erb
├── _feedback.html.erb
├── _header.html.erb
├── _large_cta.html.erb
├── _life_stages.html.erb
├── _nav.html.erb
├── _search.html.erb
├── footer
│   ├── _footer.html.erb
│   ├── _footer_menu.html.erb
│   ├── _footer_newsletter.html.erb
│   └── _footer_social.html.erb
└── svg
    └── _icon_sprite.html.erb
```
I will move this to other place which is not "shared" folder.